api/server.test.js

```diff
@@ -0,0 +1,206 @@
+const request = require('supertest');
+const server = require('./server')
+const db = require('../dbConfig')
+
+describe('Testing Server Connection and Setup', () => {
+    beforeAll(async () => {
```

Comment:
I know all this setup can be tedious but this is great practice and shows you're serious about the profession/craft 🔥